src/symmetric.jl

```diff
@@ -1052,4 +1059,4 @@ function Base.replace_in_print_matrix(A::HermOrSym,i::Integer,j::Integer,s::Abst
     ijminmax = minmax(i, j)
     inds = A.uplo == 'U' ? ijminmax : reverse(ijminmax)
     Base.replace_in_print_matrix(parent(A), inds..., s)
-end
+end
```

Comment:
```suggestion
end

```

It's common style to have an empty line at the end. No need to modify this.